translations/strings.yaml

```diff
@@ -444,3 +444,12 @@ mobileOnboarding:
 
 inAppAuth:
   changeEmailLink: Change email
+  signInButton: Sign in
+  signInSubtitle: Continue to Mozilla VPN by signing in with your Firefox account
+  forgotPasswordLink: Forgot your password?
+  passwordInputPlaceholder: Enter password
+  passwordInputLabel: Password
+  invalidPasswordErrorMessage: Invalid password. Please try again
+  signInDisclaimer:
+    value: By proceeding, you agree to the following
```

Comment:
Looks like GitHub ate my other comment when submitting the review :-( 

Do you have a screenshot of this looks like? I expect it to be problematic for several languages, because it's basically [forcing the order](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#avoid-concatenations-use-placeholders-instead), and we might need something else.

Reply:
I removed the string and associated UI so that we can explore this and address in a follow-up PR. 

Screenshot of view from spec:
<img width="227" alt="Screen Shot 2022-02-16 at 6 09 28 PM" src="https://user-images.githubusercontent.com/22355127/154378799-f677f6dd-bb25-4b4e-82da-77383c79fba8.png">

Reply:
Unfortunately this design really doesn't work for localization :-(

Reply:
Noted. Thank you for flagging! I filed [this ticket](https://mozilla-hub.atlassian.net/browse/VPN-1774) to address this. cc @peterbenvenuto @chrisrooneys